var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -53,11 +56,36 @@ def install(self, spec, prefix):
                 'TBB=-L%s -ltbb' % spec['tbb'].prefix.lib,
             ])
 
+        # --------------- Locate BLAS and LAPACK
+        exts = ['.a', '.so']
+        if ('+fpic' in spec):
+            exts.reverse()
+
+        # Find BLAS for the not-so-powerful makefile
```

Comment:
OK, I will use the interface from #657.  Then I don't think I will need to
do the testing I suggested above (which would only prove that the old code
breaks on some systems).


On Thu, Apr 28, 2016 at 10:20 AM, Denis Davydov <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/suite-sparse/package.py:
>
> > @@ -53,11 +56,36 @@ def install(self, spec, prefix):
> >                  'TBB=-L%s -ltbb' % spec['tbb'].prefix.lib,
> >              ])
> >
> > +        # --------------- Locate BLAS and LAPACK
> > +        exts = ['.a', '.so']
> > +        if ('+fpic' in spec):
> > +            exts.reverse()
> > +
> > +        # Find BLAS for the not-so-powerful makefile
>
> should not you use the interface introduced here #657
> <https://github.com/LLNL/spack/pull/657>
> that is self.spec.blas_shared_lib and alike?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/LLNL/spack/pull/859/files/72acefc39fe1d4bc299d86065119354d613009d6#r61434933>
>